ibis/backends/bigquery/__init__.py

```diff
@@ -441,11 +442,22 @@ def create_table(
         self,
         name: str,
         obj: pd.DataFrame | ir.Table | None = None,
+        *,
         schema: ibis.Schema | None = None,
         database: str | None = None,
+        temp: bool | None = None,
+        overwrite: bool = False,
     ) -> ir.Table:
         if obj is None and schema is None:
-            raise ValueError("The schema or obj parameter is required")
+            raise com.IbisError("The schema or obj parameter is required")
+        if temp is True:
+            raise com.IbisInputError(
+                "BigQuery backend does not yet support temporary tables"
+            )
+        if overwrite is not False:
+            raise com.IbisInputError(
```

Comment:
Should these be `NotImplementedError` instead?

Reply:
probably -- I think i did this to keep our exceptions within the ibis coms, but it's just confusing.  We probably need to revamp all the ibis exceptions anyway.  I'll change these.